main.tf

```diff
@@ -300,7 +306,7 @@ module "ecs_cloudwatch_sns_alarms" {
 module "alb_target_group_cloudwatch_sns_alarms" {
   source  = "cloudposse/alb-target-group-cloudwatch-sns-alarms/aws"
   version = "0.15.0"
-  enabled = var.alb_target_group_alarms_enabled
+  enabled = var.alb_enabled && var.alb_target_group_alarms_enabled
```

Comment:
```suggestion
  enabled = local.alb_enabled && var.alb_target_group_alarms_enabled
```

Reply:
Fixed this line and a couple of others I found, `var.alb_enabled` should only be referenced when setting the local variable now.